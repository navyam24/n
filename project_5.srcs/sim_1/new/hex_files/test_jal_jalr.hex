00300093
00700113
00a00193
008000ef
0000802b
00f00213
01400293
0000802b
0000007b
