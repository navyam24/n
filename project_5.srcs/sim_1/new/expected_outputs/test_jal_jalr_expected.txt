out 0 00000000000000000000000000010000 00000010 16